### PR TITLE
KK-1176 | fix: static routes and footer links

### PR DESCRIPTION
**Pull request by melniiv**
## Description
- Fix frontpage language switch click redirect to accessibility page (now it redirects to localized front page)
- Fix static page routing (isCmsPage check is added, so all the static routes now only changing the locale part of the path
- Fix footer to use headless cms navigation

## Context

[KK-1176](https://helsinkisolutionoffice.atlassian.net/browse/KK-1176)

## Screenshots

[KK-1176]: https://helsinkisolutionoffice.atlassian.net/browse/KK-1176?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr571.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by karisal-anders** [comment]
@melniiv Some test snapshots are out-of-sync as they had the hardcoded footer contents previously but now they're ending up empty. How about mocking the footer menu query response in those tests so they'll be populated there?

File: src/domain/app/footer/__mocks__/footerMenuMock.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/* eslint-disable max-len */
+import { MenuDocument } from 'react-helsinki-headless-cms/apollo';
+import { MockedResponse } from '@apollo/client/testing';
+
+import { languageToMenuNameMap } from '../constants';
+
+/**
+ * This is a mock of the response from the footer menu query.
+ * Data taken from production CMS query response.
```
[the remark this answers is not in the document]
Comment (melniiv) [reply src/domain/app/footer/__mocks__/footerMenuMock.ts]: fixed

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr571.dev.hel.ninja** :rocket::rocket::rocket:

**Review by karisal-anders — APPROVED** [review]
Tested with production CMS data, all header and footer menus seem to work and language changes changes the page to the translated one (The Terms of Service page is not yet translated in CMS but that's to be done as part of [KK-1179](https://helsinkisolutionoffice.atlassian.net/browse/KK-1179). Also profile page language change worked and the main page's language change worked.

The heading size in CMS pages has changed, it's a lot bigger than in current production:
image
So that might be still finetuned but overall looks good!

**Pull request merged**